erpnext/controllers/stock_controller.py

```
@@ -152,6 +155,62 @@ def validate_duplicate_serial_and_batch_bundle(self, table_name):
 					)
 				)
 
+	def get_item_wise_inventory_account_map(self, company):
```

Ruff (0.14.1)

Unused method argument: `company`

(ARG002)